molecule/scenario_resources/Dockerfile.j2

```diff
@@ -12,9 +12,9 @@ ENV container docker
 
 {% if item.name.startswith('centos-7') -%}
 
-RUN yum install -y epel-release ;\
-yum makecache fast ;\
-yum update -y ;\
+RUN yum install -y epel-release &&\
```

Comment:
While i strongly believe this change should be there, it seems to consistently break package upgrade tests.
@mikedep333 any thoughts?

Rationale: Without it, molecule produced improper images due to a failed dnf install and even refrained to recreate that layer in subsequent calls.

Reply:
I figured that we probably do not want do the Dockerfile preparation stuff with the upgrade tests, as those containers should have gone through this process already.

Reply:
BTW: molecule retries to build these images three times. But docker does not rebuild this layer if it thinks it was successful, thereby defeating the retry mechanism.